### ISSUE TITLE
not support submatch yet?

### ISSUE THREAD

**Issue by davidbuhtig** [XVimProject/XVim#922]
nothing happens:
:s#\(\S\)#//\1#

**Comment by 99arobe** [comment]
I don't think that matching with groups is supported yet. See #310 and #720 . It'd be a great feature.

**Comment by 99arobe** [comment]
Actually, there was a possible fix in #496 - Not sure why this isn't working currently though.
